pkgs/collection/lib/src/unmodifiable_wrappers.dart

```diff
@@ -193,4 +193,12 @@ abstract mixin class UnmodifiableMapMixin<K, V> implements Map<K, V> {
   /// operations that change the map are disallowed.
   @override
   void clear() => _throw();
+
+  /// Throws an [UnsupportedError].
+  @Deprecated('This will be removed at a later date.')
```

Comment:
Can we just use `@deprecated` if the message isn't saying anything  anyway.

Reply:
done